internal_test_data/complex_test_case_with_version.yaml

```diff
@@ -26,8 +26,6 @@ instance_provision: 'on_demand'
 
 use_mgmt: true
 mgmt_port: 10090
```

Comment:
You can remove mgmt_port as well, right?

Reply:
yes, missed it. already fixed